packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -46,126 +106,394 @@ With Html syntax lexer (enabled by default):
 </Header>
 ```
 
-### Full Header example
+## API
+
+| Prop name  | Type      | Default       | Required | Description                              |
+| ---------- | --------- | ------------- | -------- | ---------------------------------------- |
+| `class`    | `string`  | `null`        | no       | Custom CSS class                         |
+| `color`    | `string`  | `transparent` | no       | Color variant: `transparent`, `inverted` |
+| `isSimple` | `boolean` | `false`       | no       | Compact and centered version of Header   |
+
+On top of the API options, you can add `data-*` or `aria-*` attributes to
+further extend component's descriptiveness and accessibility. These attributes
+will be passed to the topmost HTML element of the component.
+
+## Supported Content
+
+To create a responsive header with top-level navigation, there are the following
+building blocks to use:
+
+1. Inside Header:
+   1. mobile-only actions, including toggle button by default,
+   2. desktop-only actions with primary and secondary action slots.
+2. Inside Header Dialog:
+   1. primary actions slot (all breakpoints),
+   2. secondary actions slot (all breakpoints).
+
+## Header
+
+### Mobile-Only Actions
+
+Slot for actions that are intended to display on mobile and tablet screens only.
```

Comment:
Mobile-Only Actions, but they are for tablet too?

Reply:
I know… A more precise name could be non-desktop actions… I'll happily accept anything better than _mobile_ 🙂.

Reply:
I understand your point, but I still think it wouldn't be much better (though it would be more precise, that's right).

I'm taking full responsibility for leaving it this way because:

- We might decide to make the breakpoint configurable in the future so being less precise is an advantage here: mobile is this, desktop is that, and you configure what's in between.
- Something like `SmallScreenActions` and `LargeScreenActions` might make even more sense. But I'm not changing it now 🙏🏻 .
- At the end of the day, tablet is a also a mobile (portable) device 🎉 .

Reply:
OK :)